tests/functional/config.py

```diff
@@ -2,12 +2,14 @@
 
 
 class TestSetting(BaseSettings):
-    CAPABILITIES = {'browser_name': 'firefox', 'selenoidoptions': {"enableVnc": True, "enableVideo": False}}
-    TEST_CAPS: dict = {
-        "browserName": "firefox",
-        "acceptInsecureCerts": True,
-        "moz:debuggerAddress": True,
-        }
+    class Firefox:
+        CAPABILITIES = {'browserName': 'firefox', 'selenoid:options': {"enableVnc": True, "enableVideo": False}}
+        # CAPABILITIES = Capabilities(browser_name='firefox', selenoidoptions={"enableVnc": True, "enableVideo": False})
+        CONTAINER: dict = {
```

Comment:
Delete this field.

Reply:
fixed in 9b20364a666f03413286df31a3d4c2cfb919f0c5